### DISCUSSION TITLE
Layout looks bad in mobile

### DISCUSSION THREAD

**Discussion by NonsensicalOne** [yhype/yhype-feedback#32] [Bug] [answered]
![Screenshot_20240802-130347](https://github.com/user-attachments/assets/175fbb0f-e1d3-441b-996c-8bb4a956f946)

there is a similar discussion to this but there weren't any screenshots shown off

**Comment by antonkomarev** [comment]
Yeah, I'm mostly using it on desktop, that's why I haven't tuned it for mobile, but want to do it in the future. Not so easy to do it right now.

**Reply by antonkomarev** [reply, marked as answer]
@NonsensicalOne 
You can try new interface, it includes dark theme and has mobile support now!

<img width="1511" height="829" alt="image" src="https://github.com/user-attachments/assets/22065a8d-688e-4184-b03e-00bfb2766bec" />

**Reply by NonsensicalOne** [reply]
thanks!
